src/issue/screens/new-issue.screen.js

```diff
@@ -106,75 +127,95 @@ class NewIssue extends Component {
       issueTitleHeight,
       issueComment,
       issueCommentHeight,
+      keyboardHeight,
     } = this.state;
 
     return (
-      <ViewContainer>
+      <KeyboardAwareContainer
+        style={{ flex: 1 }}
```

Comment:
As `new-issue` screen has been written with styled components, you'd better convert it, too.